pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -429,11 +447,17 @@ func (nim *nodeInfoManager) updateNodeInfoObject(
 		}
 	}
 
+	isMigratable := false
+	if driverIsMigratableFunc, ok := migratedDrivers[driverName]; ok {
+		isMigratable = driverIsMigratableFunc()
```

Comment:
If `driverName` is not found, should we log here as done above in https://github.com/davidz627/kubernetes/pull/2/files#diff-0e13ddc214dda9ec6a0a089c62d372aeR384 ?